gateway-messages/src/lib.rs

```diff
@@ -158,9 +173,11 @@ pub enum ResponseKind {
     SerialConsoleAttachAck,
     SerialConsoleWriteAck { furthest_ingested_offset: u64 },
     SerialConsoleDetachAck,
-    SysResetPrepareAck,
-    // There is intentionally no `SysResetTriggerAck` response; the expected
-    // "resposne" to `SysResetTrigger` is an SP reset, which won't allow for
+    PowerState(PowerState),
+    SetPowerStateAck,
+    ResetPrepareAck,
+    // There is intentionally no `ResetTriggerAck` response; the expected
+    // "resposne" to `ResetTrigger` is an SP reset, which won't allow for
```

Comment:
typo: `resposne`

Reply:
nice catch, thanks. I think I've made that typo multiple times now...